src/world/components/lib/getRandomTransition.js

```diff
@@ -0,0 +1,7 @@
+const matchTransitionFor = require('./matchTransitionFor');
+
+module.exports = function getRandomTransition(world) {
+  const moveableSet = world.things.filter(thing => thing.locations.length > 1);
+  const randomThing = moveableSet[Math.floor(Math.random() * moveableSet.length)];
+  return randomThing && [matchTransitionFor(randomThing, world.numRules, world.rules), randomThing];
```

Comment:
I think locations needs to be improved. It's not the case that things simply move from point A to point B, they also interact with the location itself! A person on a hill might gaze out at the scenery, a person in a library might pick up a book to read.